about_us.md

```diff
@@ -16,23 +16,23 @@ nav_order: 10
 
 ## Who are we?
 
-We, Medik8s team, are four Red Hatters with a strong passion to High Availabitlity (HA) solutions in the Kubernetes world.
+We, Medik8s team, are five Red Hatters with a strong passion to High Availability (HA) solutions in the Kubernetes world.
```

Comment:
I would remove the number, who cares about that?
Maybe even remove Red Hat.
"We are a small team with..."
🤔